frappe/automation/doctype/assignment_rule/assignment_rule.py

```diff
@@ -17,6 +18,8 @@ def validate(self):
 		if not len(set(assignment_days)) == len(assignment_days):
 			repeated_days = get_repeated(assignment_days)
 			frappe.throw(_("Assignment Day {0} has been repeated.".format(frappe.bold(repeated_days))))
+		if not self.role and not self.users:
+			frappe.throw(_("Please select users or a role for assignment"))
```

Comment:
Users and Role instead of "users" and role". It's Frappe standard to use capital letters when you're referring to DocTypes.